### PR TITLE
Adding Mobile Menu (pair programming)

### PR DESCRIPTION
**Pull request by omar25ahmed**
### Mobile Menu Feature:
- I modified the structure of the nav to fit the project
- the I added the style of the menu 
- after that my Partner has done the JavaScript part 
- I added the eslinter for JS language

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,19 +34,18 @@
...
-            <ul class="ul-menu">
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
-            </ul>
-          </nav>
-        </div>
+      <header class="navbar">
+        <a href="#" class="logo"><img src="images/MyLogo.png" alt="" /></a>
+        <div id="icon" class="icon fas fa-bars"></div>
```
Comment (Rustamxon7) [inline index.html:39]: **You made a great effort in creating this webpage but there are tiny issues that need to be fixed.** 

- [ ] I can see the `Hamburger` button on your desktop version of your project. The requirements says The `Hamburger` button should not be displayed on the desktop version:

Screen Shot 2022-01-25 at 7 34 51 AM

File: index.html [hunk 2]
```diff
@@ -27,19 +34,18 @@
...
+        <nav class="ul-menu">
+          <ul class="o">
+            <li class="desktop"><a href="#portfolio">Portfolio</a></li>
+            <li class="desktop"><a href="#about">About</a></li>
+            <li class="desktop"><a href="#contact">Contact</a></li>
+          </ul>
```
Comment (Rustamxon7) [inline index.html:45]: One last thing is... 
- [ ] I can barely see your `X` button when I open the Pop-Up menu. I would suggest you make it more visible to users and on your each list items have list-style `.` which is makes your Pop-Up menu different from the original design I would strongly suggest you to this as well. 

As you can see in this image...

Screen Shot 2022-01-25 at 7 37 30 AM

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Team, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi TEAM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**